README.md

```diff
@@ -62,8 +37,57 @@ To build for the host system, run:
                 If a file is highlighted, open current directory in destination panel.
 
 
-## Credits
+## Settings
+
+For instance to change the UI/font size on e.g. Steam Deck create a file called commander.cfg in ~/.config/ (also called $HOME/.config/) with these settings in it:
+
+```
+disp_autoscale_dpi=0
+disp_ppu_x=2.5
+disp_ppu_y=2.5
+```
+
+You can find other settings in this source file:
+https://github.com/od-contrib/commander/blob/079a84c8a8cc25e1899cb11bcfce8e9e2ed9fca9/config.h
+
+
+
+## Building
+
+To build for RetroFW, run:
+
+```bash
+BUILDROOT=<path to buildroot> ./build.sh retrofw
+```
+
+To build for RG350/GKD350h, run:
+
+```bash
+BUILDROOT=<path to buildroot> ./build.sh rg350
+```
+
+The package will be built at `build-retrofw/commander.ipk`.
+
+To build for the host system, run:
+
+```bash
+./build-host.sh
+```
+
+
+## DinguxCommander Credits
+
+Development: Mia<br />
+http://beyondds.free.fr<br />
+https://tardigrade-nx.github.io
+<br />
+Font: Beycan Çetin
+
+
+## OD Commander Credits
 
-Homepage:      http://beyondds.free.fr/
-Development:   Mia
-Font:          Beycan Çetin
+Homepage:<br />
+https://wiki.batocera.org/od_commander<br />
+Development: Glem M.<br />
```

Comment:
My name is Gleb, but there is no need for a credits section as it's already clear from GitHub.
That URL is not the homepage either, it's just a Batocera wiki page.
Commander is not Batocera-specific, Batocera is but one of the many distributions that use it.

Reply:
Noted. Should I remove the OD Commander credits, then?

Reply:
The original DinguxCommander credits should stay there because there is no Git history for that (it was distributed as a zip file with source code).

Reply:
Of course! - that's why the original credits are kept in all my commits.

But I removed your credit (as you requested unless I misunderstood).

Reply:
Yep that's exactly what I meant, thanks